src/player.cpp

```diff
@@ -4045,6 +4048,15 @@ void player::mod_pain(int npain) {
         if( has_trait( trait_NOPAIN ) ) {
             return;
         }
+        // always increase pain gained by one from these bad mutations
+        if ( has_trait( trait_MOREPAIN ) ) {
+            npain += std::max(1, roll_remainder( npain * 0.25f ));
+        } else if ( has_trait ( trait_MOREPAIN2 ) ){
+            npain += std::max(1, roll_remainder( npain * 0.5f ));
+        } else if ( has_trait ( trait_MOREPAIN3 ) ) {
+            npain += std::max(1, roll_remainder( npain * 1.0f ));
```

Comment:
Why multiply with a `float` - `roll_remainder` takes a `double`, so the result is converted to that anyway.

Also: when you add new code, please use the proper white spacing: no space before `(` and one space after it. See the block below at line 4060.

Reply:
Don't know why, I just used the code below.

Will fix the spaces.

Reply:
More spaces.

Reply:
There is no spaces in the std::max() calls.

Reply:
Thanks, I can't seem to see the the spaces. Must be some mental problem. Like how some people seem to notice dirt less.